src/pushNotifications/TokenHandling.ts

```diff
@@ -0,0 +1,75 @@
+import { Platform } from 'react-native';
+
+import { secrets } from '../config';
+import * as SecureStore from 'expo-secure-store';
+import * as appJson from '../../app.json';
+
+
+const namespace = appJson.expo.slug as keyof typeof secrets;
+
+export enum PushNotificationStorageKeys {
+    PUSH_TOKEN = 'PUSH_TOKEN',
+    IN_APP_PERMISSION = 'IN_APP_PERMISSION',
```

Comment:
indentation should be always 2 spaces

Reply:
didn't see eslint autofix did not catch this.
it is changed in e2f9b849f1157cec42932f4fc81cf4ce13f20fc8